test/src/test/java/org/corfudb/runtime/view/stream/BackpointerStreamViewTest.java

```diff
@@ -113,4 +111,30 @@ public void moreReadQueueTest() {
             assertThat(new String(payLoad).equals(String.valueOf(i)));
         }
     }
+
+    /**
+     *  test proper backpointer termination at the head of a stream
+     *
+     * */
+    @Test
+    public void headOfStreamBackpointerTermination() {
+
+        final int totalEntries = 101;
```

Comment:
totalEntries is  PARAMETERS.NUM_ITERATIONS_LOW (number of entries in StreamA) + 1 (number of entries in StreamB)... I can change it to be PARAMETERS.NUM_ITERATIONS_LOW  1